### PR TITLE
Fix Bedrock devnet stateviz container

### PR DESCRIPTION
**Pull request by mdehoog**
**Description**

Currently the Bedrock devnet stateviz container is failing to parse the snapshot logs, due to [this change](https://github.com/ethereum-optimism/optimism/commit/9eea16e31dea037a45cd93e8bdd6a59bb20691b6#diff-511cbf4bb175ffd3df8b49e206073b0cc80140ecfe160a92bfd253cfb5728de2R544) switching the logs JSON key from `l2SafeHead` to `l2Safe`. The `stateviz` JSON parser wasn't updated in the process, causing it to error with `failed to decode snapshot log` errors.

**Tests**

This is a bugfix, tested by browsing to http://localhost:9090 locally and checking that it's working. Although I guess we should build an integration test to ensure this doesn't regress in the future.

**Additional context**

N/A

### CODE REVIEW

File: op-node/cmd/stateviz/main.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ type SnapshotState struct {
 	L1Head          eth.L1BlockRef `json:"l1Head"`          // what we see as head on L1
 	L1Current       eth.L1BlockRef `json:"l1Current"`       // l1 block that the derivation is currently using
 	L2Head          eth.L2BlockRef `json:"l2Head"`          // l2 block that was last optimistically accepted (unsafe head)
-	L2SafeHead      eth.L2BlockRef `json:"l2SafeHead"`      // l2 block that was last derived
+	L2SafeHead      eth.L2BlockRef `json:"l2Safe"`          // l2 block that was last derived
```
Comment (mdehoog) [inline op-node/cmd/stateviz/main.go:45]: should we change the name of the variable too?

Comment (Inphi) [reply op-node/cmd/stateviz/main.go:45]: good find. Yeah, the variable name was changed to `l2Safe`.

**Review by Inphi — APPROVED** [review]
LGTM!

File: op-node/cmd/stateviz/main.go (hunk 1 above)
Comment (mdehoog) [reply op-node/cmd/stateviz/main.go]: cool, updated in #d5bad28

**Review by mslipper — APPROVED** [review]

**Comment by mslipper** [comment]
Triggered CI.

**Pull request merged**